vdb/lib/utils.py

```diff
@@ -453,7 +453,7 @@ def trim_epoch(
 def vers_compare(compare_ver: str | int | float, vers: str) -> bool:
     """Purl vers based version comparison"""
     min_version, max_version, min_excluding, max_excluding = None, None, None, None
-    if vers == "*" or compare_ver is None or compare_ver == "*":
+    if vers == "*" or compare_ver == "*" or not compare_ver:
```

Comment:
We want to avoid returning True if compare_ver is 0 or empty. So used an explicit check for None

Reply:
@prabhu Can you clarify why that is desirable? Lots of false positives or what?

Reply:
Correct. With vers check we can only return True for definite cases.

```
Reaching here without having finished the check before means that the "tested version" is NOT in the range.
```

https://github.com/package-url/purl-spec/blob/version-range-spec/VERSION-RANGE-SPEC.rst

Reply:
Could you remove this change alone. The rest can be merged.